tools/disasm/variables.txt

```diff
@@ -970,7 +970,7 @@
     0x801BDBA0:("D_801BDBA0","UNK_TYPE1","",0x1),
     0x801BDBA4:("D_801BDBA4","UNK_TYPE1","",0x1),
     0x801BDBA8:("D_801BDBA8","UNK_TYPE1","",0x1),
-    0x801BDBB0:("D_801BDBB0","u8","",0x1),
+    0x801BDBB0:("gWeatherMode","u8","",0x1),
```

Comment:
Should this change be added to namefixer or was it only accessed by code?

Reply:
I just did a double check, and it seems like the only undecompiled file left that uses it is `z_kankyo`. I checked roz's github and it doesn't look like that file has been started yet, so it should be fine to not add it?

Reply:
Probably fine then yeah